portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.time.format.DateTimeFormatter;  
+import java.time.LocalDateTime;
+import java.time.format.FormatStyle;
+
+// Class representing a comment
+public class Comment {
+    private String name;
+    private String comment;
+    private String date;
+
+    public Comment(){
+        name="";
+        comment="";
+        date="";
```

Comment:
Include spaces around the =, like: `name = "";` and so on.
(see the style guide on adding spaces around operators :https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)